releases/cloud/point-2024-08-26-release-note.md

```diff
@@ -0,0 +1,23 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.4.59.1
+---
+
+# August 26, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+## Improvements & Bug Fixes
+
+* **Improved the information message** when making changes to the Appearance and E-mail sections in Settings in order to include more details on the size and format of the logo and banner. 
+
+* **Fixed an issue** that caused the Privacy column on the Group Access report to appear as empty for some Microsoft Groups and Teams. 
+
+* **Improved the functionality** of policies by implementing a timeout to limit the possibility of a policy running indefinitely. 
```

Comment:
ne bih rekao da se tu poboljšala funkcionalnost, samo smo job napravili otpornijim na probleme u velikim okolinama
suggestion:
Improved the Policy monitor job, which is responsible for vulnerability detection. Timeouts were implemented to prevent the job from running indefinitely in case a single policy detection runs longer than expected due to a high number of workspaces in the tenant.